### DISCUSSION TITLE
Applied Physics Question Paper - HPTU Exam Helper

### DISCUSSION THREAD

**Discussion by giscus** [Eniv-Studios/hptu_exam_helper_forum#3] [General]
# Applied Physics Question Paper - HPTU Exam Helper

HPTU Exam Helper is a website aimed at assisting students of B.Tech CSE at Himachal Pradesh Technical University (HPTU) by providing access to syllabus, study materials, question papers.

https://www.hptuexamhelper.in/question_papers/applied_physics_ques_paper

**Comment by sadhika-puri122** [comment]
great

**Reply by Shubham-EV** [reply]
Credits goes to you.🏅
